example_config.yaml

```diff
@@ -0,0 +1,78 @@
+# Top-level parameters
+
+exp-name: E-MTAB-1234
+droplet: False
+
+# Matrices, slots correspond to keys in .layers, unless .X or .raw.X, which
+# which case those are interpreted as explicit top-level slots in the anndata.
+
+matrices:
+  - slot: raw.X
+    measure: counts
+    cell_filtered: True
+    gene_filtered: False
+    normalised: False
+    log_transformed: False
+
+  - slot: filtered
+    measure: counts
+    cell_filtered: True
+    gene_filtered: True
+    normalised: False
+    log_transformed: False
+    
+    parameters:
+        min_count: 3
+  
+  - slot: normalised
+    measure: counts
+    cell_filtered: True
+    gene_filtered: True
+    normalised: True
+    log_transformed: True
+    
+
+# Cell groups, slots correspond to columns in .obs. Two categories:
+#     - curated: present before analysis. These will be output for curation.
+#     - analysis: derived from analysis: e.g. clusters.  
+# 
+# Cell groupings marked with 'markers: True' will have those markers written to
+# the bundle
+
+cell_groups:
+  - slot: organism_part
+    kind: curation
+    markers: False
+
+  - slot: louvain_resolution_0.7
+    kind: analysis
+    markers: True
+    parameters:
+      alorithm: louvain
+      resolution: 0.7
+      
+  - slot: louvain_resolution_1.0
+    kind: analysis
+    markers: True
+    parameters:
+      alorithm: louvain
+      resolution: 1.0
+
+# Dimension reductions, slots correspond to keys in .obsm
+
+dimension_reductions:
+ - slot: X_umap_neighbors_n_neighbors_3
```

Comment:
Some minor indentation difference here compared to the previous sections.

Reply:
Yup, just a prototype, hadn't spend much time on the tidying yet